ssl/d1_lib.c

```diff
@@ -369,7 +384,12 @@ int dtls1_handle_timeout(SSL *s)
         return 0;
     }
 
-    dtls1_double_timeout(s);
+    if (s->d1->timer_cb != NULL) {
+        const unsigned ms = s->d1->timer_cb(s, s->d1->timeout_duration_ms);
+        s->d1->timeout_duration_ms = ms;
+    }
+    else
+        dtls1_double_timeout(s);
```

Comment:
The correct style would be:
````
} else {
    dtls1_double_timeout(s);
}
````
If one part of an if...else... uses {} then they all do.